web/client/plugins/querypanel/AttributeFilter.jsx

```diff
@@ -0,0 +1,369 @@
+/**
```

Comment:
This is a connected copy of `web/client/components/data/query/GroupField.jsx`. Please reuse the component